builder/dockerfile/builder.go

```diff
@@ -229,6 +229,12 @@ func (b *Builder) build(source builder.Source, dockerfile *parser.Result) (*buil
 			return nil, errdefs.InvalidParameter(errors.Errorf("failed to reach build target %s in Dockerfile", b.options.Target))
 		}
 		stages = stages[:targetIx+1]
+		// Add `LABEL` command if `--label` option specified
```

Comment:
We should use this for the case when `--target` is not set as well and remove `addNodesForLabelOption`.

Reply:
OK, I will remove the `addNodesForLabelOption` and use `NewLabelCommand` to add the `--label` options into the last stage, thus we will have the `LABEL` there.